config/selectionCfg_MuMu_UL17.cfg

```diff
@@ -0,0 +1,187 @@
+[selections]
+baseline = isLeptrigger && pairType == 3 && nleps == 0 && nbjetscand > 1
+baseline_boosted = pairType == 3 && nleps == 0 && isLeptrigger && isBoosted==1
+
+btagM        = (bjet1_bID_deepFlavor > 0.3040 && bjet2_bID_deepFlavor < 0.3040) || (bjet1_bID_deepFlavor < 0.3040 && bjet2_bID_deepFlavor > 0.3040) #only 1 jet with medium btag working point
+btagLL       = bjet1_bID_deepFlavor > 0.0532 && bjet2_bID_deepFlavor > 0.0532 #both jets with loose btag working point
+btagMM       = bjet1_bID_deepFlavor > 0.3040 && bjet2_bID_deepFlavor > 0.3040 #both jets with medium btag working point
+
+# H-bbtag requirements - ParticleNet  
+pNetBTagH = fatjet_particleNetMDJetTags_score >= 0.987
+pNetBTagM = fatjet_particleNetMDJetTags_score >= 0.9714
+pNetBTagL = fatjet_particleNetMDJetTags_score >= 0.9105
+
+massCut    = bH_mass > 50 && bH_mass < 270 && tauH_mass > 20 && tauH_mass < 130
+massCutInv = (bH_mass < 50 || bH_mass > 270) && (tauH_mass < 20 || tauH_mass > 130)
+
+SR       = isOS != 0 && dau1_iso < 0.15 && dau2_iso < 0.15
+
+baselineMcut    = baseline, massCut
+baselineInvMcut = baseline, massCutInv
+
+res1b = baseline, btagM , isBoosted != 1, massCut
+res2b = baseline, btagMM, isBoosted != 1, massCut
+
+boostedL_pnet = baseline_boosted, pNetBTagL
+boostedM_pnet = baseline_boosted, pNetBTagM
+boosted_semi   = !(pairType == 3 && nleps == 0 && isLeptrigger && isBoosted == 1 && fatjet_particleNetMDJetTags_score >= 0.9105), baseline, btagLL, isBoosted==1, fatjet_hasMatchedSj==1, massCut
```

Comment:
`!(pairType == 3 && nleps == 0 && isLeptrigger && isBoosted == 1 && fatjet_particleNetMDJetTags_score >= 0.9105)` could be summarized in `fatjet_particleNetMDJetTags_score < 0.9105`, all the other requirements are requested right after

Reply:
Can we please keep it like this? Simona tested this expression and it works. These line are more subtle than they appear and it's a category that we won't use in any case, it's there just for testing.

Reply:
Fine for me.